roles/add-vhost/tasks/main.yml

```diff
@@ -35,22 +34,4 @@
     force_basic_auth: yes
     body_format: json
     body:
-      domain_name: "{{ vhost }}{{ stanford_environment }}.cardinalsites.stanford.edu"
-  when:
-    vhost is defined
-
-# Ex. drupal7-update-status.cardinalsites.stanford.edu.
-- name: Add inventory_hostname as domain
```

Comment:
Why is this being dropped?

Reply:
This task adds `inventory_hostname` as a custom domain, so `g2scd7.cardinalsites.stanford.edu`.

Reply:
I remember why this is in there.

If you migrate a site without a vhost (e.g., if `supri-b` didn't have a vhost), supri-b.cardinalsites.stanford.edu would return a 404, because `acsf_site_name` is `suprib`and users would have no way of knowing that